src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1718,6 +1718,16 @@ void GraphOptimizer::FuseConvolutionSumAndConvolutionSumActivation(Graph &graph)
         if (mergedConv->isConstant() && !sum->isConstant())
             continue;
 
+        auto checkInputCompatibility = [](NodePtr sumNode) {
```

Comment:
@dmitry-gorokhov do we still need `Convolution::getSumMemDesc` update when we add this logic?

Reply:
Yes, because each component have to provided consistent and safe behavior indepandently. 
Convolution responsibility is to correctly execute fused Sum operation for any shapes. GraphOptimizer responsibility is to avoid Conv+Sum fusing for cases there we expect bad performance.